apps/wallet/src/ui/app/hooks/useCreateAccountMutation.ts

```diff
@@ -69,10 +69,12 @@ export function useCreateAccountsMutation() {
 				type === 'mnemonic-derived' &&
 				validateAccountFormValues(type, accountsFormValues, password)
 			) {
-				await backgroundClient.unlockAccountSourceOrAccount({
-					password,
-					id: accountsFormValues.sourceID,
-				});
+				if (password) {
```

Comment:
curious why this check was added since we have `validateAccountFormValues` above and shouldn't be able to submit the form without a password?

Reply:
For mnemonic accounts it is possible to create one if the account source is already unlocked. So password can be empty for those cases